Shared/Objects/AppNotifications.swift

```diff
@@ -0,0 +1,14 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Foundation
+
+enum AppNotifications {
+    // Posted when the now playing data has likely become stale.
+    static let nowPlayingDidChange = Notification.Name(rawValue: "nowPlayingDidChange")
```

Comment:
Don't create a new object to hold notification definitions, that's already in `SwiftfinNotifications.swift`.